pkg/controller/elasticsearch/client/model.go

```diff
@@ -104,97 +103,35 @@ type ClusterStateNode struct {
 	} `json:"attributes"`
 }
 
-// Shards contains the shards in the Elasticsearch routing table
-// mapped to their shard number.
-type Shards struct {
-	Shards map[string][]Shard `json:"shards"`
-}
+// Shards contains the shards in the Elasticsearch cluster
+type Shards []Shard
 
-// ClusterState partially models Elasticsearch cluster state.
-type ClusterState struct {
-	ClusterName  string                      `json:"cluster_name"`
-	ClusterUUID  string                      `json:"cluster_uuid"`
-	Version      int                         `json:"version"`
-	MasterNode   string                      `json:"master_node"`
-	Nodes        map[string]ClusterStateNode `json:"nodes"`
-	RoutingTable RoutingTable                `json:"routing_table"`
+// Shard partially models Elasticsearch cluster shard.
+type Shard struct {
+	Index  string `json:"index"`
+	Shard  string `json:"shard"`
+	Prirep string `json:"prirep"`
```

Comment:
Nit: going by the docs prirep and state have known values, so it might be worthwhile to make these more specific types: https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-shards.html